app/services/imports/sales_logs_import_service.rb

```diff
@@ -214,6 +214,17 @@ def rescue_validation_or_raise(sales_log, attributes, previous_status, exception
         @logs_overridden << sales_log.old_id
         attributes.delete("income1")
         save_sales_log(attributes, previous_status)
+      elsif sales_log.errors.of_kind?(:equity, :over_max) || sales_log.errors.of_kind?(:equity, :under_min)
+        @logger.warn("Log #{sales_log.old_id}: Removing equity as the equity is invalid")
+        @logs_overridden << sales_log.old_id
+        attributes.delete("equity")
+        save_sales_log(attributes, previous_status)
+      elsif sales_log.errors.of_kind?(:postcode_full, :wrong_format)
+        @logger.warn("Log #{sales_log.old_id}: Removing postcode as the postcode is invalid")
+        @logs_overridden << sales_log.old_id
+        attributes.delete("postcode_full")
+        attributes["pcodenk"] = attributes["la"].present? ? 1 : nil
```

Comment:
I think we should keep this kind of logic in the main block — it will be easy to miss, otherwise.

Reply:
What do you mean the main block? This is only meant to set it if the postcode is cleared as a result of validation, cause then we'd want to set it to not known  and get the la persisted instead